File: vademecum/code-src/zmienne_i_struktury_danych/Tablice.js

var t = ['a b c', 8, 3, 2];

console.log(
	"pierwszy element: " + t[0] + "\n" +
	"długość tablicy: " + t.length + "\n"
);

// usuwamy 2 elementy od pozycji 1
t.splice(1, 2)

// dodajemy nowe elementy
t[5] = "pp";
t[7] = "ww";

// wypisanie wszystkich elementów
for (var i=0; i<t.length; ++i) {
	console.log("t["+ i +"]="+ t[i] +"\n");
}

// za pomocą for..in
for (var ii in t) {
	console.log("t["+ ii +"]="+ t[ii] +"\n");
}

File: vademecum/code-src/zmienne_i_struktury_danych/Tablice.c

#include <stdio.h>

int main() {
	int tablica[3]; // tablica 3 elementowa liczb całkowitych
	
	tablica[0] = 13; // do elementów tablicy odwołujemy się z użyciem [],
	                 // elementy tablicy indeksujemy od zera
	tablica[2] = 17;
	
	printf("%d %d\n", tablica[0], tablica[1]);
	// nie zainicializowane elementy (tak samo jak nie zainicjalizowane zmienne)
	// będą mieć przypadkową wartość (nie jest to jednak dobre źródło losowości)
}

void xxx(int n) {
	// Język C od wersji C99 pozwala na korzystanie z tablic zmiennej długości (*VLA*),
	// czyli tablic których rozmiar nie jest stałą czasu kompilowania a zmienną - np.:
	float vals[n];
	v[0] = 21;
	/* ... */
}
